include/mimir/formalism/factories.hpp

```diff
@@ -348,6 +351,12 @@ GroundAtomList<P> PDDLFactories::get_ground_atoms_from_indices(const Iterable& a
     return result;
 }
 
+template<PredicateCategory P>
```

Comment:
There is already the function `get_factory` that provides access to all pddl objects, including GroundAtoms. Can we use it instead?

Reply:
Sure, do you want the get_ground_atoms() function to be removed on the c++ side altogether or merely change it to use the factory instead?